client/src/components/AlertBanner.js

```diff
@@ -19,26 +18,22 @@ function SlideTransition(props: SlideProps) {
 }
 
 export default function AlertBanner() {
-  const { alert, setAlert } = useContext(AppContext);
+  const { alert, setAlert } = useAlert();
   const alertType = {};
 
-  function alertSeverity() {
-    switch (alert){
-      case 'ExerciseSaved':
-        alertType.severity = 'success';
-        alertType.msg = 'Exercise successfully saved.';
-        break;
-      case 'ErrorOccurred':
-        alertType.severity = 'error';
-        alertType.msg = 'An unexpected error occurred. Please try again.';
-        break;
-      default:
-        return false;
-    }
+  switch (alert){
+    case 'ExerciseSaved':
+      alertType.severity = 'success';
+      alertType.msg = 'Exercise successfully saved.';
+      break;
+    case 'ErrorOccurred':
+      alertType.severity = 'error';
+      alertType.msg = 'An unexpected error occurred. Please try again.';
+      break;
+    default:
+      return false;
```

Comment:
I'm confused by this `return`. It will cause the banner to not display anything. Maybe that's what you want, but you should at least log something so you can tell what happened when the banner doesn't show up.